src/components/settings/SettingsSSOTab/ExistingSSOConfigs.jsx

```diff
@@ -126,7 +143,16 @@ const ExistingSSOConfigs = ({
                       {!config.was_valid_at && (
                         <div className="d-flex">
                           <Dropdown.Item
-                            onClick={() => deleteConfig(config.id)}
+                            onClick={() => {
```

Comment:
provider data can be a list now, so we have to loop over all of them and delete each